test/fixtures/profiles/fetcher-failures/git-bad/inspec.yml

```diff
@@ -0,0 +1,9 @@
+name: git-bad
+license: Apache-2.0
+summary: A profile with a bad git dependency
+version: 0.1.0
+supports:
+  platform: os
+depends:
+  - name: nonesuch
+    git: http://localhost/no/such
```

Comment:
If `localhost` is intended to be invalid, it should be `localhost.invalid`. `localhost` is generally a valid resolvable host. There are a few other uses of localhost in this PR this applies to as well.

https://tools.ietf.org/html/rfc2606

Reply:
Thanks for the tip, I've modified the hostnames accordingly.